### ISSUE TITLE
PNG or JPG set for the body {background) overlays an absolutely positioned div tag declared later

### ISSUE THREAD

**Issue by GoogleCodeExporter** [liudonghua123/ie7-js#30]
```
This is a new issue with the latest beta. the problem does not occur with
IE7_0_9

Without the ie7 fix, a certain page displays fine: a "top navigation menu"
(absolutely positioned) shows as normal over a background image (png or
jpg) declared for the body.

When the newest ie7 script is included, the browser ie6 renders the
background over the topnav menu div tag. the menu appears briefly but then
is hidden.

the css is:

body {
    margin: 0 auto;
    background: #bbb url(../images/interface/background_gradient.jpg) repeat-x
left top;
}


and

#topnav {
    position: absolute;
    width: 600px;
    padding: 0;
    margin: 0;
    top: 120px;
    right:  0px;
    overflow: visible;
    text-align: right;
}

```

Original issue reported on code.google.com by `pgur...@gmail.com` on 9 Feb 2008 at 10:06

**Comment by GoogleCodeExporter** [comment]
```
> Without the ie7 fix, a certain page displays fine

DO you have a sample test page?
```

Original comment by `dean.edw...@gmail.com` on 21 Feb 2008 at 9:56

**Comment by GoogleCodeExporter** [comment]
```
I do - let me anonymize the client's urls and such...
```

Original comment by `pgur...@gmail.com` on 21 Feb 2008 at 10:05

**Comment by GoogleCodeExporter** [comment]
```
Here is the public sample:  http://64.13.230.81/ie7test/
```

Original comment by `pgur...@gmail.com` on 5 Mar 2008 at 7:55

**Comment by GoogleCodeExporter** [comment]
```
Where can I get version IE7_0_9 to revert back to a version that doesn't have 
this bug?
```

Original comment by `jyear...@kwiktrip.com` on 17 Mar 2008 at 3:27

**Comment by GoogleCodeExporter** [comment]
```
https://sourceforge.net/project/showfiles.php?group_id=109983&package_id=119707&
release_id=350483
Beware that this is alpha software and does not work as well as v2.0b3. You'll 
find
install instructions here:
http://dean.edwards.name/IE7/install.html

```

Original comment by `svar...@gmail.com` on 20 Mar 2008 at 9:05

**Comment by GoogleCodeExporter** [comment]
```
I was poking around for a different solution and I found another script that 
also
fails in this very same manner. 

http://www.twinhelix.com/css/iepngfix/

However, this new script generates a somewhat-helpful error message:  

IEPNGFix: Children of positioned element are unclickable

After this, the background image then overlays the foreground text like Dean's 
script
does.

Does this point you in the right direction for a fix?


```

Original comment by `pgur...@gmail.com` on 4 May 2008 at 6:03

**Comment by GoogleCodeExporter** [comment]
```
No fix for this yet?
```

Original comment by `ejcor...@gmail.com` on 5 Jun 2009 at 4:45

**Comment by GoogleCodeExporter** [comment]
```
This should be fixed in the next release (2.1).
```

Original comment by `dean.edw...@gmail.com` on 10 Feb 2010 at 9:21
- Changed state: **Fixed**
